presto-docs/src/main/sphinx/ext/download.py

```diff
@@ -17,41 +17,56 @@
 # noinspection PyUnresolvedReferences
 from sphinx.errors import SphinxError
 
+TD_LINK_SERVER = 'http://www.teradata.com/presto'
+TD_LINK_CLI = 'http://www.teradata.com/presto'
+TD_LINK_JDBC = 'http://www.teradata.com/presto'
+TD_LINK_DEMO_CDH = 'http://www.teradata.com/presto'
+TD_LINK_DEMO_HDP = 'http://www.teradata.com/presto'
+TD_LINK_PRESTO_DOCS = 'http://www.teradata.com/presto'
+TD_LINK_PRESTO_YARN_PACKAGE = 'http://www.teradata.com/presto'
+TD_LINK_PRESTO_AMBARI_PLUGIN = 'http://www.teradata.com/presto'
+TD_LINK_SLIDER = 'http://archive.apache.org/dist/incubator/slider/0.80.0-incubating/slider-assembly-0.80.0-incubating-all.tar.gz'
+TD_LINK_ODBC_MAC = 'http://www.teradata.com/presto'
+TD_LINK_ODBC_WIN64 = 'http://www.teradata.com/presto'
+TD_LINK_ODBC_WIN32 = 'http://www.teradata.com/presto'
+TD_LINK_ODBC_LINUX64 = 'http://www.teradata.com/presto'
+TD_LINK_ODBC_LINUX32 = 'http://www.teradata.com/presto'
+TD_LINK_ODBC_DOCS = 'http://www.teradata.com/presto'
+
 GROUP_ID = 'com.facebook.presto'
 ARTIFACTS = {
-    'server': ('presto-server', 'tar.gz', None),
-    'cli': ('presto-cli', 'jar', 'executable'),
-    'jdbc': ('presto-jdbc', 'jar', None),
-    'verifier': ('presto-verifier', 'jar', 'executable'),
-    'benchmark-driver': ('presto-benchmark-driver', 'jar', 'executable'),
-}
-
-
-def maven_filename(artifact, version, packaging, classifier):
-    classifier = '-' + classifier if classifier else ''
-    return '%s-%s%s.%s' % (artifact, version, classifier, packaging)
-
-
-def maven_download(group, artifact, version, packaging, classifier):
-    base = 'https://repo1.maven.org/maven2/'
-    group_path = group.replace('.', '/')
-    filename = maven_filename(artifact, version, packaging, classifier)
-    return base + '/'.join((group_path, artifact, version, filename))
+    'server': ('presto-server-rpm-0.127t', 'rpm', TD_LINK_SERVER),
```

Comment:
Though to answer your question, all of the download links have been changed already.  I'm closing this pr since it' pushed already.  But if you want to add comments to what was actually added, it's just the final commit (Teradata additions on top of 0.141 release)